test/parallel/test-global.js

```diff
@@ -61,6 +61,11 @@ for (const moduleName of builtinModules) {
     'navigator',
   ];
   assert.deepStrictEqual(new Set(Object.keys(global)), new Set(expected));
+  expected.forEach((value) => {
+    if (typeof global[value] === 'function') {
+      assert.strictEqual(global[value].name, value);
+    }
```

Comment:
Maybe something like that?
```suggestion
    const desc = Object.getOwnPropertyDescriptor(global, value);
    if (typeof desc.value === 'function') {
      assert.strictEqual(desc.value.name, value);
    } else if (typeof desc.get === 'function') {
      assert.strictEqual(desc.get.name, `get ${value}`);
    }    
```

Reply:
done